model/project.go

```diff
@@ -71,6 +71,10 @@ type Project struct {
 
 	// Number of includes in the project cached for validation
 	NumIncludes int `yaml:"-" bson:"-"`
+
+	// EmptyTaskSelectors stores task selectors that don't target any tasks for a build variant.
+	// This is only for validation purposes.
+	EmptyTaskSelectors map[string][]string `yaml:"-" bson:"-"`
```

Comment:
I think it's best to keep public just to avoid writing accessor methods (this is defined in the `model` package but used also in the `validator` package